unpub/lib/src/app.dart

```diff
@@ -89,6 +97,16 @@ class App {
     return req.requestedUri.resolve(reference).toString();
   }
 
+  /// validate the bearer token in the request headers against the stored token
+  Future<void> _validateOpaqueToken(shelf.Request request) async {
```

Comment:
Match the opaqueToken (if passed) against the request authentication header value